src/components/Question.tsx

```diff
@@ -13,12 +13,20 @@ import { Question as QuestionType } from "../types";
 
 type Props = {
     question: QuestionType;
-    onSubmit: (answer: string) => void;
+    onSubmit: (answer: string[]) => void;
     compact?: boolean;
 };
 
 const Question = ({ question, onSubmit, compact = false }: Props) => {
-    const [selectedAnswer, setSelectedAnswer] = React.useState<string | null>(null);
+    const [selectedAnswer, setSelectedAnswer] = React.useState<string[]>([]);
```

Comment:
Maybe we should rename this to `selectedAnswers` to be more clear that it can hold multiple answers